sinch/domains/voice/models/calls/responses.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import Price
+
+
+@dataclass
+class GetVoiceCallResponse(SinchBaseModel):
+    from_: str
+    to: dict
```

Comment:
This property can be defined as an object with 2 properties `type` and `endpoint` as it is done below for the properties `user_rate` and `debit` and their model `Price`